src/main/pages/che-7/contributor-guide/ref_che-theia-plug-in-api.adoc

```diff
@@ -8,6 +8,7 @@ folder: che-7/contributor-guide
 summary:
 ---
 
+:page-liquid:
 
 [id="che-theia-plug-in-api_{context}"]
```

Comment:
I don't get this one and a few other modules like this. Where is the context defined or inherited from? :( @rkratky @themr0c 

It's headings render with an anchor like this `#theia-namespace_{context}`...

Reply:
@MichalMaler could you document somewhere where `:page-liquid:` should be defined and what it does? For example, in the contrib draft in our devtools gdrive. Ping me if you can't find it.

Reply:
@boczkowska 

> could you document somewhere where :page-liquid: should be defined and what it does?

See https://github.com/asciidoctor/jekyll-asciidoc#enabling-liquid-preprocessing for background.

> I don't get this one and a few other modules like this. Where is the context defined or inherited from? :( @rkratky @themr0c

Every assembly should set its context, and modules should be included from assemblies.

> It's headings render with an anchor like this #theia-namespace_{context}...

That's a mistake (because an ID like that is useless and won't compile with ccutil). It is most likely caused by one of the following:
* The parent assembly doesn't set context properly.
* The module is used as stand-alone (i.e. not included from any assembly), but it doesn't set its context. So, it should either define its own context (like assemblies do), or be included from some assembly.

Reply:
>That's a mistake (because an ID like that is useless and won't compile with ccutil). It is most likely caused by one of the following:
I filed an issue for it, see my comment down the thread. This is obviously a mistake but we should deal with it in a separate PR.